### PR TITLE
Use Lukas' workspace diagram.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: src/tech/yk_structure.md [hunk 1]
```diff
@@ -16,14 +16,17 @@ workspace then talks to the internal workspace via `extern` functions defined
...
-       External Workspace             |       Internal Workspace
---------------------------------------------------------------------
-
-interpreter --\
-              |
-              +---> ykshim_client --------> ykshim --> JIT runtime
-              |
-tests  -------/
+          External Workspace                Internal Workspace
+
```
Comment (ptersilie) [inline src/tech/yk_structure.md:20]: Maybe we can remove some of these extra lines?

File: src/tech/yk_structure.md [hunk 2]
```diff
@@ -16,14 +16,17 @@ workspace then talks to the internal workspace via `extern` functions defined
...
++------------------------------------+ +-----------------------------+
+|                                    | |                             |
```
Comment (ptersilie) [inline src/tech/yk_structure.md:22]: This one too.

File: src/tech/yk_structure.md [hunk 3]
```diff
@@ -16,14 +16,17 @@ workspace then talks to the internal workspace via `extern` functions defined
...
+| Interpreter +                      | |                             |
+|             |                      | |                             |
+|             +----> ykshim_client +----> yk_shim +----> JIT runtime |
+|             |                      | |                             |
+|       Tests +                      | |                             |
+|                                    | |                             |
```
Comment (ptersilie) [inline src/tech/yk_structure.md:28]: And this.

**Comment by vext01** [comment]
Like that?

**Comment by ptersilie** [comment]
Looks good!

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**